packages/action-group/README.md

```diff
@@ -308,3 +308,69 @@ The `justified` attribute will cause the `<sp-action-group>` element to fill the
     </sp-action-button>
 </sp-action-group>
 ```
+
+## Sizes
```

Comment:
Can we do this like these: https://github.com/adobe/spectrum-web-components/pull/2561/files#diff-cf4b0c014149d7962740bdc8188c60f4e4f91b5c9f4c1ad195bd94b2ad9d7a26R61-R97 in place of these: https://github.com/adobe/spectrum-web-components/blob/main/packages/action-group/README.md?plain=1#L27-L44